docs/Using-Fleet/Vulnerability-Processing.md

```diff
@@ -83,6 +83,7 @@ As of right now, the following distributions are supported:
 - Ubuntu
 - RHEL based distros (Red Hat, CentOS, Fedora, and Amazon Linux)
 
+As of right now, only app names with all-ASCII characters are supported. Apps with names featuring Non-ASCII characters such as Cyrillic will not generate matches. 
```

Comment:
```suggestion
As of right now, only app names with all ASCII characters are supported. Apps with names featuring Non-ASCII characters, such as Cyrillic, will not generate matches. 
```

Reply:
I wonder if we should make this statement a bit more broad. Even if we fix  https://github.com/fleetdm/fleet/issues/7067, we still won't be able to match these software. The NVD CPE database mainly contains English software. We won't be able to search for vulnerabilities for non-English software.

Reply:
hmmm I think this is more a limitation of NVD in general, not of our own vuln system, right? I don't think we should document their limitations here. Maybe we would discuss NVD limitations in a blog post about it in a way that's more ephemeral. But putting it in the docs means we risk it going out of date.